src/authorization/jwt/MsJwtProvider.ts

```diff
@@ -52,24 +60,30 @@ export const authorizeJwt = (
       compliant = jwtProp === policy[key];
     }
 
-    Logger.debug(
-      `isValidJwtToken: ${key}, expected: ${policy[key]}, found: ${jwtProp}, ${compliant}`,
+    Logger.info(
+      `isValidJwtToken: ${key}, expected: ${policy[key]}, found: ${jwtProp}, ${compliant}`, logContext
```

Comment:
DEBUG is only for debugging. There are too many Logger.info calls because debug was broken. I fixed this in the new PR: 
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/221

There is minimal info logging for the endpoints. In principal just the name of request and the result of the endpoint. With debug flag you see all details